src/MSALWrapper.Test/AuthFlowTest/AuthFlowBrokerTest.cs

```diff
@@ -396,6 +398,27 @@ public async Task BrokerAuthFlow_GetTokenInteractive_OperationCanceledException_
             authFlowBroker.ErrorsList[2].Message.Should().Be("Interactive Auth (with extra claims) timed out after 15 minutes.");
         }
 
+        /// <summary>
+        /// Ensure <see cref="IPCAWrapper.WithPromptHint"/> be invoked in <see cref="TokenFetcherPublicClient.GetTokenNormalFlowAsync"/>.
+        /// </summary>
+        /// <returns>The <see cref="Task"/>.</returns>
+        [Test]
+        public async Task GetTokenNormalFlowAsync_GetTokenInteractive_WithPromptHint()
+        {
+            this.SilentAuthUIRequired();
+            this.InteractiveAuthResult();
+
+            this.MockAccount();

```

Comment:
This part isn't present in #11, @shalinikhare27. Can you shed some light on why this is necessary?

Reply:
I need to MockAccount to mock the method `TryToGetCachedAccountAsync` to get account. You can see the `MockAccount` implementation here and every test in the file uses this method as well. :)